ui/src/shared/components/RefreshingView.tsx

```diff
@@ -168,15 +162,13 @@ const mstp = (state: AppState, ownProps: OwnProps): StateProps => {
     state,
     ownProps.dashboardID
   )
-  const timeRange = getTimeRangeByDashboardID(state, ownProps.dashboardID)
-
   const valuesStatus = getDashboardValuesStatus(state, ownProps.dashboardID)
-
+  const {properties} = ownProps
+  const timeRange = getActiveTimeRange(ownProps.timeRange, properties.queries)
   const timeZone = state.app.persisted.timeZone
 
   return {
-    endTime: getEndTime(timeRange),
-    startTime: getStartTime(timeRange),
+    ranges: timeRange,
```

Comment:
let's keep calling this timeRange to signal to the next programmer that this is the same old timeRange that they know from the rest of the app.

Reply:
but we have `timeRange` in the `OwnProps` :(

Reply:
why are we passing the same prop to the same component twice? What is the difference between them? Which one is redundant?

Reply:
The difference between them is that the `timeRange` refers to the dropdown selection while the `ranges` refer to the specific ranges in the query script editor

Reply:
This is not correct.